RefRed/initialization/gui.py

```diff
@@ -124,13 +131,22 @@ def set_main_table(self):
                     _widget.setChecked(False)
                     _widget.setEnabled(True)
 
-                    _signal_func = lambda state=0,row=row_index: self.parent.reduction_table_visibility_changed_test(state, row)
+                    _signal_func = lambda state=0, row=row_index: self.parent.reduction_table_visibility_changed_test(  # noqa: E501, E731
```

Comment:
Out of curiosity, is `# noqa: E501, E731` the trick that will allow me to use lambda func in the future?